code/src/GridAPI/Pagination.ts

```diff
@@ -65,15 +65,26 @@ namespace GridAPI.Pagination {
      *
      * @export
      * @param {string} gridID Id of the Grid from which to obtain the pagination Index
-     * @return {*}  {number} Index of the current page, 0 based. Return -1 if the Grid is not yet initialized.
+     * @return {*}  {string} Stringified JSON structure containing Index of the current page, error message and code, and success boolean
      */
-    export function GetCurrentPage(gridID: string): number {
+    export function GetCurrentPage(gridID: string): string {
```

Comment:
the return should be `number | string`

Reply:
We are returning a JSON stringify, why would need number?